### ISSUE TITLE
[Filter Panel]: QA: Screen reader focus stucks after activating close button in Filter overlay

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9398]
### Description

Page Element:
Filter overlay: Close button

Issue Description:
After activating the close button, the overlay disappears but the Screen reader focus gets stuck hence we are unable to navigate back or forward using (Left/Right swipe). We have to touch on a random spot on the screen to be able to navigate again.

Expected:
After closing the overlay, the user should be able to navigate back and forward smoothly with "Left/Right swipe)

### Component(s) impacted

Filter Panel

### Browser

Safari

### Carbon for IBM.com version

v1.38.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

Web component Canary- https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components Web component Staging- https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by andy-blum** [comment]
Closing as outdated, this has not appeared in recent screen reader sessions.
